parquet-column/src/main/java/org/apache/parquet/filter2/recordlevel/FilteringRecordMaterializer.java

```diff
@@ -98,4 +99,17 @@ public void skipCurrentRecord() {
   public GroupConverter getRootConverter() {
     return rootConverter;
   }
+
+  // The following two methods are kept for backward compatibility
+  public static List<Integer> getIndexFieldPathList(PrimitiveColumnIO c) {
```

Comment:
As these methods were removed in a previous path I guess, they shall be deprecated (if not used internally). Could you please deprecate them?

Reply:
Sure.